client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEdit.tsx

```diff
@@ -138,27 +110,27 @@ export const PrescriptionLineEdit: React.FC<PrescriptionLineEditModalProps> = ({
 
   const canAutoAllocate = !!(currentItem && draftPrescriptionLines.length);
 
-  const handleSave = async (onSaved: () => boolean | void) => {
-    if (
-      getAllocatedQuantity(draftPrescriptionLines) === 0 &&
-      !showZeroQuantityConfirmation
-    ) {
-      setShowZeroQuantityConfirmation(true);
-      return;
-    }
-
-    try {
-      await onSave();
-      setIsDirty(false);
-      if (!!placeholder) {
-        const infoSnack = info(t('message.placeholder-line'));
-        infoSnack();
-      }
-      return onSaved();
-    } catch (e) {
-      // console.error(e);
-    }
-  };
+  // const handleSave = async (onSaved: () => boolean | void) => {
+  //   if (
+  //     getAllocatedQuantity(draftPrescriptionLines) === 0 &&
+  //     !showZeroQuantityConfirmation
+  //   ) {
+  //     setShowZeroQuantityConfirmation(true);
+  //     return;
+  //   }
+
+  //   try {
+  //     await onSave();
+  //     // setIsDirty(false);
+  //     if (!!placeholder) {
+  //       const infoSnack = info(t('message.placeholder-line'));
+  //       infoSnack();
+  //     }
+  //     return onSaved();
+  //   } catch (e) {
+  //     // console.error(e);
+  //   }
+  // };
```

Comment:
This is some of the logic needed for UI error indications, so just leaving here commented out for now so we don't forget about it or lose it.